.gitignore

```diff
@@ -1,4 +1,6 @@
-/binding.node
-/node_modules/
-/package-lock.json
-/target/
+node_modules
+package-lock.json
+target
```

Comment:
```suggestion
node_modules/
package-lock.json
target/
```
Otherwise they also match files with that name. Probably also applies to .npmignore.